### PR TITLE
add dev command + defaults

### PR DESCRIPTION
**Pull request by craigmulligan**

### CODE REVIEW

File: app/main.js [hunk 1]
```diff
@@ -21,8 +22,18 @@
         "URL_LAUNCHER_OVERLAY_SCROLLBARS": process.env.URL_LAUNCHER_CONSOLE == null ? 0 : process.env.URL_LAUNCHER_CONSOLE === '1'
     };
 
-    let window = null;
+    if (process.env.NODE_ENV === 'development') {
+      console.log('Running in development mode')
+      electronConfig = Object.assign(electronConfig, {
```
Comment (stefan-mihaila) [inline app/main.js]: `Object.assign` modifies the object in place, no reason to use its return value. It's good practice to use `const` anyway, so in https://github.com/resin-io/resin-electronjs/pull/32 I made `electronConfig` a constant.

File: app/main.js [hunk 2]
```diff
@@ -21,8 +22,18 @@
         "URL_LAUNCHER_OVERLAY_SCROLLBARS": process.env.URL_LAUNCHER_CONSOLE == null ? 0 : process.env.URL_LAUNCHER_CONSOLE === '1'
     };
 
-    let window = null;
+    if (process.env.NODE_ENV === 'development') {
+      console.log('Running in development mode')
```
Comment (stefan-mihaila) [inline app/main.js]: For consistency, let's add a trailing semicolon.

File: app/package.json [hunk 3]
```diff
@@ -5,7 +5,8 @@
   "main": "main.js",
   "scripts": {
     "start": "NODE_ENV=production startx ./node_modules/electron/dist/electron .",
-    "test": "NODE_ENV=development startx ./node_modules/electron/dist/electron ."
```
Comment (stefan-mihaila) [inline app/package.json]: I know you just left it the way it was, but I think `NODE_ENV` should be `test` here.

File: app/main.js [hunk 4]
```diff
@@ -21,8 +22,18 @@
...
+      electronConfig = Object.assign(electronConfig, {
+        "URL_LAUNCHER_URL": 'file:///' + path.join(__dirname, 'data', 'index.html'),
+        "URL_LAUNCHER_HEIGHT": 600,
```
Comment (stefan-mihaila) [inline app/main.js]: Let's drop the unnecessary quotes. I'll merge https://github.com/resin-io/resin-electronjs/pull/32, you can just rebase or merge and `npm run lint` to have some sane code style.

**Review by stefan-mihaila — CHANGES_REQUESTED** [review]
I merged https://github.com/resin-io/resin-electronjs/pull/32, please update PR to make the linter happy and we're 👍

File: app/package.json (hunk 3 above)
Comment (craigmulligan) [reply app/package.json]: Yea I was unsure if that'd break anything? Would this even test anything @curcuz probably should just remove it?

File: app/main.js (hunk 4 above)
Comment (craigmulligan) [reply app/main.js]: Oh nice! Will do 👍

**Comment by craigmulligan** [comment]
@stefan-mihaila updated.

**Review by stefan-mihaila — APPROVED** [review]
Looks good to me 👍

**Pull request merged**